public/distinctions.json

```
@@ -0,0 +1,23 @@
+{
+  "Paris Peace Forum": {
+    "description": {
+      "fr": "Sélectionné comme solution par le Forum de Paris sur la paix 2020",
+      "en": "Selected as Solution by the Paris Peace Forum 2020"
+    },
+    "link": "https://parispeaceforum.org/projects/community-against-disinfo/"
+  },
+  "Nobel Prize Summit": {
+    "description": {
+      "fr": "Présenté comme une «innovation de source ouverte pour l'intégrité de l'information» au sommet du prix Nobel 2023",
+      "en": "Featured as “Open-Source Innovation for Information Integrity” at the Nobel Prize Summit 2023"
+    },
+    "link": "https://digitalpublicgoods.net/information-pollution/"
+  },
+  "Digital Public Good": {
+    "description": {
+      "fr": "Reconnu comme bien public numérique par le PNUD, l'UNICEF et le Norad",
```

“Norad” is the abbreviation of the “Norwegian agency for development cooperation”, which is “agence” in French, female word, so I'd argue Norad is female too 🙂 
```suggestion
      "fr": "Reconnu comme bien public numérique par le PNUD, l'UNICEF et la Norad",
```